docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:14.04
+MAINTAINER svanoort <samvanoort@gmail.com>
```

Comment:
@oleg-nenashev Do you have a link to the discussion?  There's not enough here to make licensing worthwhile probably, but I wanted to put someone as a point of contact for the images in case they get (ab)used beyond the original use cases.  A "guilty party" if you will ;-)